openpype/hosts/maya/plugins/publish/extract_playblast.py

```diff
@@ -43,7 +43,12 @@ def _capture(self, preset):
                     json.dumps(preset, indent=4, sort_keys=True)
                 )
             )
-
+        if (
+            preset["viewport_options"].get("reloadTextures")
+            and preset["viewport_options"].get("textures")
+        ):
+            lib.reload_textures()
+            preset.pop("reloadTextures")  # not supported by `capture`
```

Comment:
You should always POP even if `reloadTextures` is present but `textures` is False so it needs to be out of that if statement.
```suggestion
        preset.pop("reloadTextures", None)  # not supported by `capture`
```
By adding `None` as second argument you allow it to not exist.